ParallaxGenLib/src/ParallaxGenDirectory.cpp

```diff
@@ -155,13 +156,19 @@ auto ParallaxGenDirectory::mapFiles(const unordered_set<wstring> &NIFBlocklist,
       WinningSlot = NIFUtil::getSlotFromTexType(WinningType);
     }
 
+    bool IgnoreTexture = false;
+    if (isFileInBSA(Texture, BSAExcludes) && (WinningSlot == NIFUtil::TextureSlots::PARALLAX)) {
+      spdlog::trace(L"Ignoring height map {} from Vanilla BSA", Texture.wstring());
```

Comment:
Follow the same convention for logging here as below
```suggestion
      spdlog::trace(L"Mapping Textures | Mapping Failed | Texture: {} | Type: {} | Ignoring height map {} from Vanilla BSA", Texture.wstring());
```